code/modules/power/cable.dm

```diff
@@ -678,7 +678,7 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe("cable restrai
 			if (showerror)
 				to_chat(user, "<span class='warning'>You can only lay cables on catwalks and plating!</span>")
 			return
-		if(U.intact)						//can't place a cable if it's a plating with a tile on it
+		if(U.underfloor_accessibility < UNDERFLOOR_HIDDEN)//can't place a cable if it's a plating with a tile on it
```

Comment:
isn't nothing less than this?

Reply:
Theres already a `T.underfloor_accessibility < UNDERFLOOR_INTERACTABLE` check above, so this is just duplicate code, even on our current master. If the underfloor is not accessible(which is connected to t-ray scanning and whatnot) the sanity check will already hit it.

Just gonna 86 this check.